Applications/DataExplorer/VtkAct/VtkCustomInteractorStyle.h

```diff
@@ -72,13 +72,13 @@ public slots:
 	virtual ~VtkCustomInteractorStyle();
 
 	/// @brief The vtk object to pick.
-	vtkDataObject* Data;
+	vtkDataObject* _data;
 
 	/// @brief The mapper for highlighting the selected cell.
-	vtkDataSetMapper* selectedMapper;
+	vtkDataSetMapper* _selectedMapper;
 
 	/// @brief The actor for highlighting the selected cell.
-	vtkActor* selectedActor;
+	vtkActor* _selectedActor;
```

Comment:
I don't think so. Whenever the variables are used, VTK expects "normal" pointers, so I'd have to use ```.get()``` which makes the whole concept debatable in my opinion. On top of that I get error messages from stdlib when the unique_ptr should be deleted but VTK still uses the memory. If it's possible to do it, it's at least not straightforward.